src/twisted/topfiles/9139.bugfix

```diff
@@ -0,0 +1 @@
+A patch to handle when a client provides an unspported authentication key type.
```

Comment:
This should be in the form of "<fqpn> now <x>" or so, such as "twisted.conch.ssh.userauth.SSHUserAuthServer now gracefully handles unsupported authentication key types."